features/step_definitions/setup_steps.rb

```diff
@@ -213,3 +221,28 @@ def service_status_public (service_id)
   service_status_public("1123456789012353")
   service_status_public("1123456789012354")
 end
+#@wip-create a buyer brief
+def create_buyer_brief (brief_name,framework_slug,lot,user_id)
+  file = File.read("./fixtures/briefs-DOS.json")
+  brief_data = JSON.parse(file)
+  brief_data["briefs"]["title"] = brief_name
+  brief_data["briefs"]["location"] = "Scotland"
+  brief_data["briefs"]["frameworkSlug"] = framework_slug
+  brief_data["briefs"]["lot"] = lot
+  brief_data["briefs"]["userId"] = user_id
+
+  response = call_api(:get, "/briefs", params: {user_id: user_id})
+  # puts response.body
+  JSON.parse(response.body)["briefs"].each do |brief|
+    puts brief["title"]
+  end
+  # if response.
+  # if response.code == 404
+  #   response = call_api(:post, "/users", payload: user_data)
+  #   response.code.should be(201), response.body
+  # end
```

Comment:
It's a little untidy now but that bit of code is not quite production ready and the commented stuff is to remind me of what it was before.